config/routes.rb

```diff
@@ -1,2 +1,6 @@
 Rails.application.routes.draw do
+    get 'ingredients' => 'ingredients#index'
+    resources :ingredients
+    get 'ingredient_families' => 'ingredient_families#index'
+    resources :ingredient_families
```

Comment:
A good practice is to only create routes for actions you have. In this controller you have only index and show. If you just create `resources :ingredient_families` - it will create routes for every action, even `delete` and `create`, but you don't have them in the controller.
To avoid that do it like that: `resources :ingredient_families, only: [:index, :show]`